### ISSUE TITLE
Create Facility model

### ISSUE THREAD

**Issue by andrea-williams** [bcgov/cas-registration#1320]
#### Describe the task

~~Blocked on https://github.com/bcgov/cas-registration/issues/1402~~
For Registration Part 2, we need to create a new model called `Facility`. 

#### Acceptance Criteria

- [x] create new Django migration to add Facility model (this should be built off of our existing `OperationAndFacilityCommonInfo` model)
- [x] create through-tables wherever necessary
- [x] once this PR is merged into `develop`, notify the Giraffe team so that they're unblocked

#### Additional context

- See this spreadsheet for Facility fields: https://bcgov.sharepoint.com/:x:/r/teams/00608-CPRInternal/_layouts/15/Doc.aspx?sourcedoc=%7B8CF87100-9F8D-43C1-9D1D-79C69FA259E8%7D&file=Registration%20pt.%202%20Fields.xlsx&action=default&mobileredirect=true
    - some of the fields in this file might not be finalized - reach out to Jairo if questions arise
- no need for UI work as part of this ticket
- omit geographic coordinates from model for now - requires further discussion re: PostGIS

**Comment by Sepehr-Sobhani** [comment]
Should this data model live in `registration` app? or `common` app?

**Comment by andrea-williams** [comment]
@Sepehr-Sobhani  I would suggest `registration` (at least for the time being) to keep it consistent with our `Operation` and `Operator` models.
